Quaver.Shared/IPC/QuaverIpcHandler.cs

```diff
@@ -54,7 +62,78 @@ public static void HandleProtocolMessage(string message)
                 if (game?.CurrentScreen is EditScreen screen)
                     screen.GoToObjects(message);
                 else
-                    NotificationManager.Show(NotificationLevel.Warning, "You must be in the editor to use this function!");
+                    NotificationManager.Show(NotificationLevel.Warning,
+                        "You must be in the editor to use this function!");
+            }
+            else if (message.StartsWith("map/"))
+            {
+                var game = (QuaverGame) GameBase.Game;
+                if (game.CurrentScreen.Type != QuaverScreenType.Select)
+                {
+                    NotificationManager.Show(NotificationLevel.Warning,
+                        "You must be in the song select screen to select a map!");
```

Comment:
I guess main menu should also work for this (open song select automatically), maybe some other screens? Download?